geodepy/transform.py

```diff
@@ -506,6 +506,60 @@ def conform14(x, y, z, to_epoch, trans):
     return xtrans, ytrans, ztrans
 
 
+def mga94to20(zone, east, north, ell_ht=False):
```

Comment:
To be consistent we should call this mga94to20**20**

Reply:
Refactored as requested 👍